sig-scalability/charter.md

```diff
@@ -0,0 +1,104 @@
+individual performance SLIs/SLOs, ensuring they are all oriented on user
+experience and consistent with each other:
+  - [SLIs/SLOs](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md)
+- Ensuring that each official Kubernetes release satisfies all scalability and
+performance related requirements, as stated in "Kubernetes scalability" definition.
+- Establishing and documenting best practises on how to design and/or implement
+Kubernetes features in scalable and performant way. Educating contributors and
+consulting individual designs/implementations to ensure that those are widely used.
+Example artifacts:
+  - [Scalability governance](https://github.com/kubernetes/community/blob/master/sig-scalability/governance)
+- Finding system bottlenecks and coordinating improvement on cross-cutting
+architectural changes.
+
+### Out of scope
+
+- Improving performance/scalability of features falling into charters of
+individual SIGs.
+
+
+## What can we do/require from other SIGs
+Scalability and performance are horizontal aspects of the system - changes in a
+single place of Kubernetes may affect the whole system. As a result, to
+effectively ensure Kubernetes scales, we need a special cross-SIG privileges.
+
+- We can rollback any merged PR if it has been identified as a cause of any
+  [performance/scalability SLOs] regression. The offending PR should only be
+  merged again after proving to pass  tests at scale.
+- In the even of a performance regression, we can block all PRs from being
+  merged into the relevant repos until the cause of the regression is
+	identified and mitigated.
+  The “Rules of engagement” of pausing merge-queue and rationale for
+  necessity of its introduce are explained in a separate doc. <br/>
+  TODO(wojtek-t, shyamjvs): Write it down and link here.
+- We require significant changes (in terms of impact, such as: update of etcd,
+  update of Go version, major architectural changes, etc.) may only be merged:
+  - with an explicit approval from a SIG-scalability tech lead and
+  - after having passed performance testing on biggest supported clusters (unless
+    found unnecessary by approver)
+  TODO(wojtek-t): Create a larger group (whose members doesn't have to necessary
+  be SIG scalability members) that has powers to approve such changes.
+- We can block a feature from transitioning:
+  - to Beta status, if (when turned on) it causes violation of already existing
+    performance/scalability SLOs;
+  - to GA status, when it can be used scale. That means:
+    - in rare cases, introducing a new SLI and SLO and ensuring it is met at scale
+    - in most of cases, extending scalability tests to use it and ensuring that
+      existing SLOs are still met
+- We can require a SIG to introduce a regression-catching benchmark test for a
+  scalability-critical functionality.
+
+For the record, by regression above we mean a regression identified by the set 
```

Comment:
might just want to define taxonomy at the beginning or end, this seems weirdly placed.  Could just remove.